### ISSUE TITLE
Feature request: don't ask for date and time

### ISSUE THREAD

**Issue by juef17** [mrehkopf/sd2snes#145]
My sd2snes battery seems to be dead as I'm prompted to enter the date and time every time I boot it. I don't have a use for those at the moment, so I don't intend to change the battery just to get rid of a dialog (not just because I am lazy, but mostly for environmental reasons). Would it be a reasonable feature request to have a config option to skip said dialog?

**Comment by mrehkopf** [comment]
Yeah, absolutely :) Originally the "USE_BATT" jumper on the board was intended for that. But a software options seems more practical.
